src/services/wallet.service.ts

```diff
@@ -6,7 +6,7 @@ const NON_CONTRACT_WALLET_CODE = '0x'; // any multi signature wallet is actually
 const checkIfWalletIsContract = async ({ provider, walletAddress }: { provider: EIP1193Provider; walletAddress: string }) => {
   const wrapperProvider = new ethers.providers.Web3Provider(provider);
   const res = await wrapperProvider.getCode(walletAddress);
-  return res === NON_CONTRACT_WALLET_CODE;
+  return res !== NON_CONTRACT_WALLET_CODE;
```

Comment:
I think it is confusing. The name of the function is checkIfWalletIsContract but it will return true if it isn't a contract wallet?

Reply:
res !== NON_CONTRACT_WALLET_CODE = multi sign wallet
res === NON_CONTRACT_WALLET_CODE = regular wallet